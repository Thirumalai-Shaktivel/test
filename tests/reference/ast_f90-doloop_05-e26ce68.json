{
    "basename": "ast_f90-doloop_05-e26ce68",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/../integration_tests/doloop_05.f90",
    "infile_hash": "5a491890522af8d37e521baf4bf396da8bab25c2981ac91fcb4be0c0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-doloop_05-e26ce68.stdout",
    "stdout_hash": "fe27ba59f008974b41a11006804b550fd9fa8f705fe58fc9c54c2fe8",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}