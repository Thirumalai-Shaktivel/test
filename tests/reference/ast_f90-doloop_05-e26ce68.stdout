program doloop_05
implicit none
integer :: i, j
1 j = 0
2 i_loop: do i = 1, 10
    3 if (i == 2) then
        cycle
    end if
    4 j = j + i
end do i_loop
6 if (j /= 53) then
    error stop
end if
end program doloop_05
